src/pytorch_lightning/__about__.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 import time
 
-# __version__ = "1.7.0dev"
+# __version__ = "1.7.0"
```

Comment:
it's commented.. should we remove it?

Reply:
Must be there for a reason. Doesn't matter for this PR.

Reply:
sorry, no reason this time :)

Reply:
Will do it in the PR that adds the new changelog format after release. I can't push any new changes to the branch because I accidentally opened it under release/* which is under protected rules.